show/main.py

```diff
@@ -1715,8 +1715,7 @@ def policer(policer_name, verbose):
 def sflow(ctx):
     """Show sFlow related information"""
     if ctx.invoked_subcommand is None:
-        db = Db()
-        show_sflow_global(db.cfgdb)
+        show_sflow_global(ctx.obj.cfgdb)
```

Comment:
why not use passdb?

Reply:
using pass_db decorator now in show/sflow. Also rebased, so forced pushed. 
BTW for config/main.py, I have not updated the code to use pass_db, because let's do it in separate commit. This PR is to test the current config code then in some other PR, I will change config/main.py to use pass_db plus the change in test code. Thx.